generate-assets/src/lib.rs

```diff
@@ -23,6 +24,8 @@ pub struct Asset {
     pub description: String,
     pub order: Option<usize>,
     pub image: Option<String>,
+    #[serde(rename = "crate")]
+    pub crate_name: Option<String>,
```

Comment:
Doesn't this make the link de facto `Optional`? You only need at least one of `{ crate_name, link }`

Reply:
Yes, it has to be optional at least for now, since many assets are not crates, and even some of the already-submitted crates don't have a crates.io upload. For example `bevy_doryen`, which has no crates.io upload, but there's a _different_ crate also named `bevy_doryen` that _does_ have a crates.io upload.